types/node/http.d.ts

```diff
@@ -226,8 +226,12 @@ declare module "http" {
     class Agent {
         maxFreeSockets: number;
         maxSockets: number;
-        sockets: any;
-        requests: any;
+        sockets: {
+            [key: string]: Socket[];
```

Comment:
Maybe readonly?

Reply:
Gotta be honest, most of the stuff in the node typings should be `readonly`, even the methods...
I will add it but it's really hard to keep track of those.